src/VBA-IDE-Code-Export.package/customUI/customUI.xml

```diff
@@ -0,0 +1,35 @@
+<!--RibbonX Visual Designer 1.94 for Microsoft Excel 14.0. XML Code produced on 2017/02/22-->
```

Comment:
Does this comment serve any purpose? I know it was automagically generated, but can it be removed?